charts/azuremonitor-containers/templates/omsagent-daemonset.yaml

```diff
@@ -113,6 +113,12 @@ spec:
         - mountPath: /etc/omsagent-secret
           name: omsagent-secret
           readOnly: true
+        {{- if and (.Values.Azure.proxySettings.isProxyEnabled) (.Values.Azure.proxySettings.proxyCert) }}
+        - mountPath: /etc/ssl/certs/proxy-cert.crt
```

Comment:
please add telemetry to track how many clusters are providing ca cert.

Reply:
> please add telemetry to track how many clusters are providing ca cert.

Yep, good idea to have telemetry for the cert. let me add it.